src/backend/marsha/core/tests/test_views_lti_video.py

```diff
@@ -248,7 +248,7 @@ def test_views_lti_video_student_live_mode_on(
             playlist__lti_id="course-v1:ufr+mathematics+00001",
             playlist__title="foo bar",
             playlist__consumer_site=passport.consumer_site,
-            live_state=IDLE,
```

Comment:
the test you added for idle has disappeared?

Reply:
Yes there is no reason to keep it, it returns `None` for the resource and this is what we test in `test_lti_utils`.

Reply:
Here we want to test the resource content, not if there is a resource a not depending on its state and its readyness. WDYT ?

Reply:
ok :+1: